core/templates/dev/head/services/assets-backend-api.service.ts

```diff
@@ -16,6 +16,7 @@
  * @fileoverview Service to serve as the interface for fetching and uploading
  * assets from Google Cloud Storage.
  */
+import $ from 'jquery';
```

Comment:
Why is this added? (Add a comment above.)

Reply:
Done.

Reply:
@marianazangrossi, is this comment resolved?

Also, I don't see where is this `$` used? (How it was working fine earlier though?)

Reply:
Sorry about that @DubeySandeep I missed this part. Now the comment is added.